src/app/actions/actions.ts

```diff
@@ -0,0 +1,15 @@
+"use server";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export const fetchRecipes = async () => {
+  try {
+    return prisma.recipe.findMany();
+  } catch (error) {
+    //TODO: Add some kind of toast maybe?
+    alert(error);
```

Comment:
```suggestion
    console.error(error);
```
to not get triggered by annoying alerts